packages/taquito/package.json

```diff
@@ -87,36 +87,36 @@
     "rxjs": "^6.6.3"
   },
   "devDependencies": {
-    "@babel/types": "7.16.0",
+    "@babel/types": "7.18.4",
     "@types/bluebird": "^3.5.36",
     "@types/estree": "^0.0.50",
     "@types/jest": "^26.0.23",
-    "@types/node": "^17.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.7.0",
-    "@typescript-eslint/parser": "^5.7.0",
+    "@types/node": "^17.0.42",
+    "@typescript-eslint/eslint-plugin": "^5.28.0",
+    "@typescript-eslint/parser": "^5.28.0",
     "buffer": "^6.0.3",
     "colors": "^1.4.0",
     "coveralls": "^3.1.1",
     "cross-env": "^7.0.3",
-    "eslint": "^8.5.0",
+    "eslint": "^8.17.0",
     "jest": "^26.6.3",
     "jest-config": "^26.6.3",
-    "lint-staged": "^12.1.2",
+    "lint-staged": "^13.0.1",
```

Comment:
could you confirm whether the pre commit hook was still running when you did a `git commit`?

Reply:
I havent noticed anything different from my usual but i'll make a dummy branch to be sure